tools/github-team-user-store/GitHubTeamUserStore/GitHubTeamUserStore/Program.cs

```diff
@@ -7,20 +8,62 @@ internal class Program
     {
         static async Task Main(string[] args)
         {
+            var azureBlobAccountNameOption = new Option<string>
```

Comment:
Any reason these need to be split apart? I was expecting just a blob uri. As in `string blobUri = $"https://{DefaultStorageConstants.DefaultAzureBlobAccountName}.blob.core.windows.net/{DefaultStorageConstants.DefaultAzureSdkWriteTeamsContainer}/{DefaultStorageConstants.DefaultAzureSdkWriteTeamsBlobName}";`

Reply:
If you need to have the different pieces of the uri you can use https://learn.microsoft.com/en-us/dotnet/api/azure.storage.blobs.bloburibuilder.-ctor?view=azure-dotnet to parse it into the pieces.

Reply:
The next commit will be taking the blob URI in from the command line and using the BlobUriBuilder